#this is the message received from the standard communication node for the act3d
uint32 sys_time
float32[] pos
float32[] vel
float32[] acc
float32[] force
